front/client/service/CategoryItem.ts

```diff
@@ -1,10 +1,26 @@
 import axios from 'axios'
+import { awaitExpression } from 'babel-types';
 
 class CategoryItemService {
   async getCategoryItem(): Promise<ICategoryItem[]> {
     const { data } = await axios.get('api/category/sth6l9jl', {})
     return data
   }
+
+  async addItem({ category, name }: ICategoryItem['items']) {
+    const { data } = await axios.post('api/item/sth6l9jl', null, {
+      params: {
+        category,
+        name
+      }
+    })
+    return data
+  }
+  
+  async deleteItem({ id }: ICategoryItem['items']) {
```

Comment:
Wrong type, it should be just "ICategoryItem"

Reply:
fixed